python/graphstorm/gconstruct/utils.py

```diff
@@ -182,6 +182,26 @@ def worker_fn(worker_id, task_queue, res_queue, user_parser):
     except queue.Empty:
         pass
 
+def update_two_phase_feat_ops(phase_one_info, ops):
+    """ Update the ops for the second phase feat processing
+
+    Parameters
+    ----------
+    phase_one_info: dict
+        A dict mapping file index to node/edge features info corresponding to ops.
+    ops: dict of FeatTransform
+        The operations run on the node/edge features of the node/edge files.
+    """
+    feat_info = {}
+    for i, finfo in phase_one_info.items():
```

Comment:
what is `i`? is it a sequence number?

Reply:
i is fileno. See https://github.com/awslabs/graphstorm/blob/main/python/graphstorm/gconstruct/utils.py#L243-L244 and https://github.com/awslabs/graphstorm/blob/main/python/graphstorm/gconstruct/construct_graph.py#L198